Gemfile

```diff
@@ -3,17 +3,15 @@
 source 'https://rubygems.org'
 
 gem 'ruby-termios', '>= 0.9' if RbConfig::CONFIG['target_os'] !~ /mswin|mingw|cygwin/i and RUBY_ENGINE == 'ruby'
-# This is commented out because wdm does not currently support Ruby 2.2
-#group :development do
-#  gem 'wdm', '>= 0.1.0', :platforms => [:mswin, :mingw]
-#end
+
```

Comment:
I think wdm is an old dependency. We don't seem to be using it anywhere.

Reply:
yes this can be removed.  It made file listening work better on windows.